keps/sig-cli/0031-kustomize-integration.md

```diff
@@ -167,30 +256,151 @@ To get the detailed objects in a kustomization
 kubectl get -f <dir> --dry-run -o yaml|json
 ```
 
-#### delete
+##### delete
 The command visible to users is
+
 ```
 kubectl delete -f <dir>
 ```
 
-### Implementation Details/Notes/Constraints
+#### Why we like this approach
+
+- It is capable of friction that requires deeper integration - such as producing errors referencing line
```

Comment:
Did you really mean "capable of friction," or should that have been something like, "It removes friction," or "It is capable of reducing friction?"

Reply:
+1